build.sbt

```diff
@@ -1,45 +1,74 @@
+import ReleaseTransformations._
+
 val Versions = new {
-  val akka24    = "2.4.12" // First version cross-compiled to 2.12
-  val akka25    = "2.5.0"
-  val lagom13   = "1.3.0"
-  val lagom14   = "1.4.0-M2"
-  val play25    = "2.5.0"
-  val play26    = "2.6.0"
-  val scala211  = "2.11.11"
-  val scala212  = "2.12.3"
-  val scalaTest = "3.0.1"
+  val akka24            = "2.4.12" // First version cross-compiled to 2.12
+  val akka25            = "2.5.0"
+  val lagom13           = "1.3.0"
+  val lagom14           = "1.4.0-M2"
+  val play25            = "2.5.0"
+  val play26            = "2.6.0"
+  val scala211          = "2.11.11"
+  val scala212          = "2.12.3"
+  val scalaTest         = "3.0.1"
+  val serviceLocatorDns = "2.2.2"
 }
 
 lazy val sharedSettings = Vector(
+  resolvers += Resolver.bintrayRepo("hajile", "maven"),
   organization := "com.lightbend.rp",
   organizationName := "Lightbend, Inc.",
   startYear := Some(2017),
   licenses += ("Apache-2.0", new URL("https://www.apache.org/licenses/LICENSE-2.0.txt")),
-  scalaVersion := Versions.scala212,
-  crossScalaVersions := Vector(scalaVersion.value, Versions.scala211),
-  scalacOptions ++= Vector("-deprecation")
-)
-
-lazy val scala211Settings = Vector(
   scalaVersion := Versions.scala211,
-  crossScalaVersions := Vector(Versions.scala211)
+  scalacOptions ++= Vector("-deprecation"),
+  homepage := Some(url("https://www.lightbend.com/")),
+  developers := List(
+    Developer("lightbend", "Lightbend Contributors", "", url("https://github.com/typesafehub/reactive-lib"))
+  ),
+  sonatypeProfileName := "com.lightbend.rp",
+  scmInfo := Some(ScmInfo(url("https://github.com/typesafehub/reactive-lib"), "git@github.com:typesafehub/reactive-lib.git")),
+  scalaVersion := Versions.scala211,
+  publishTo := Some(
+    if (isSnapshot.value)
+      Opts.resolver.sonatypeSnapshots
+    else
+      Opts.resolver.sonatypeStaging
+  ),
+  releasePublishArtifactsAction := PgpKeys.publishSigned.value,
+  releaseCrossBuild := false,
+  releaseProcess := Seq[ReleaseStep](
+    checkSnapshotDependencies,
+    inquireVersions,
+    runClean,
+    releaseStepCommandAndRemaining("+test"),
+    setReleaseVersion,
+    commitReleaseVersion,
+    tagRelease,
+    releaseStepCommandAndRemaining("+publishSigned"),
+    setNextVersion,
+    commitNextVersion,
+    pushChanges
+  )
 )
 
-lazy val reactiveLib = (project in file("."))
-  .enablePlugins(AutomateHeaderPlugin)
-  .settings(sharedSettings)
-  .settings(name := "reactive-lib")
+lazy val root = (project in file("."))
   .aggregate(
     common,
     akka24,
     akka25,
+    common,
```

Comment:
I think `common` is already added in the first line.

Reply:
Thanks - fixed.